multisig-account/README.md

```diff
@@ -0,0 +1,57 @@
+# Multisig Account
+
+By default accounts have a single private key,
+and that private key alone is allowed to sign transactions on behalf of an account.
+
+On Hedera, accounts natively support the ability to have multiple private keys,
+each of which is allowed to sign on behalf of the account.
+The account determines how many of its constituent keys need to sign a transaction
+for it the transaction to be considered valid.
+This is known as the signature threshold.
+
+## Code
+
+See `multisig-1-of-2.js`.
+
+We start with tasks that can be done without network interactions:
+
+- First, we generate a new ED25519 key, and a new ECDSA key.
+  These are private keys, and they will be used to create a 1-of-2 multisig account.
+- Next, we create a `KeyList` that represents a 1 of 2 multisig threshold.
```

Comment:
Curious if the ongoing conversation between what a KeyList is and a threshold key is could affect the explanation here.

Reply:
Yeah in Hedera SDK, `KeyList` represents both a `KeyList` and a `ThresholdKey` as defined in the HAPI protobuf definitions, which is where the potential confusion stems from.
I'm going to keep it simple here by sticking exclusively to the names used in the SDK,
since a developer looking at this code snippet is unlikely to peek at the protobuf definitions at this stage.
(Good on them if they do though!)